.gitignore

```diff
@@ -31,4 +31,5 @@ cover
 *.mmdb
 .DS_Store
 *.tar.gz
-.vscode
\ No newline at end of file
+.vscode
+teamocil
```

Comment:
what's that

Reply:
It's what I use to manage my local tmux environment for the project. 

https://github.com/remiprev/teamocil

It's pretty nice. From the root of the project I simply run

```sh
bin/teamocil nate
```

...and presto I have a tmux session with several windows/panes running for my dev environment. Each has the correct version of Erlang & Elixir & sources `.env` etc...

- vim
- repl
- phoenix server
- console
- etc...

I've actually committed my teamocil configs on other repos as it helped other team members learn about the app environment & get setup themselves (even when not using tmux/teamocil).

_Not sure it's appropriate to commit on this project though._

Reply:
Might suggest you throw that in your local global gitignore in that case.

Reply:
Considered it but I have about 7 other projects that have it committed.

Reply:
Sure, but does everyone use that tool? I don't; its a good case for your local global gitignore if its something you use on every project.